components/org.wso2.is.password.reencrypt/pom.xml

```diff
@@ -0,0 +1,174 @@
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>${maven-bundle-plugin.version}</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-package>org.wso2.is.password.reencrypt.internal</Private-package>
+                        <Import-Package>
+                            org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}",
+                            org.osgi.service.component; version="${osgi.service.component.imp.pkg.version.range}",
+                            org.apache.commons.logging; version="${version.commons.logging}",
+                            org.wso2.carbon.identity.core.*; version="${carbon.identity.framework.version.range}",
+                            org.apache.commons.lang; version="${commons-lang.wso2.osgi.version.range}",
+                            org.wso2.carbon.base.api,
+                            com.google.gson, javax.*, org.apache.axiom.om.*,
+                            org.wso2.carbon.base, org.wso2.carbon.consent.mgt.core.util, org.wso2.carbon.core,
+                            org.wso2.carbon.core.util, org.wso2.carbon.identity.base,
+                            org.wso2.carbon.identity.claim.metadata.mgt, org.wso2.carbon.registry.core.service,
+                            org.wso2.carbon.user.api, org.wso2.carbon.user.core.*, org.wso2.carbon.utils.dbcreator
+                        </Import-Package>
+                        <Export-Package>
+                            org.wso2.is.password.reencrypt.secondaryUserStore.*,
+                            !org.wso2.is.password.reencrypt.secondaryUserStore.internal
+                        </Export-Package>
+                    </instructions>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-scr-plugin</artifactId>
+                <version>1.22.0</version>
+                <executions>
+                    <execution>
+                        <id>generate-scr-scrdescriptor</id>
+                        <goals>
+                            <goal>scr</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <properties>
```

Comment:
any specific reason to have the properties inside components pom? usually all the properties will be in the parent pom